extensions/typescript-language-features/src/typescriptServiceClient.ts

```diff
@@ -564,27 +564,30 @@ export default class TypeScriptServiceClient extends Disposable implements IType
 		return this.executeImpl(command, args, {
 			isAsync: false,
 			token,
-			expectsResult: true
+			expectsResult: true,
+			lowPriority: true
 		});
 	}
 
 	public executeWithoutWaitingForResponse(command: string, args: any): void {
 		this.executeImpl(command, args, {
 			isAsync: false,
 			token: undefined,
-			expectsResult: false
+			expectsResult: false,
+			lowPriority: false
 		});
 	}
 
 	public executeAsync(command: string, args: Proto.GeterrRequestArgs, token: vscode.CancellationToken): Promise<any> {
 		return this.executeImpl(command, args, {
 			isAsync: true,
 			token,
-			expectsResult: true
+			expectsResult: true,
+			lowPriority: false
 		});
 	}
 
-	private executeImpl(command: string, args: any, executeInfo: { isAsync: boolean, token?: vscode.CancellationToken, expectsResult: boolean }): Promise<any> {
+	private executeImpl(command: string, args: any, executeInfo: { isAsync: boolean, token?: vscode.CancellationToken, expectsResult: boolean, lowPriority: boolean }): Promise<any> {
```

Comment:
[nit] Just make the `lowPriority` property and argument here optional. That way you don't need to touch `executeWithoutWaitingForResponse` or `executeAsync`

Reply:
Done